test/e2e/scenarios/windows/scenario.go

```diff
@@ -5,6 +5,25 @@ import (
 	"github.com/microsoft/retina/test/e2e/framework/types"
 )
 
+func ValidateWinBpfMetricScenario() *types.Scenario {
+	name := "Validate Windows BPF Basic and Advanced Metrics"
+	steps := []*types.StepWrapper{
+		{
+			Step: &ValidateWinBpfMetric{
+				KubeConfigFilePath:        "./test.pem",
+				RetinaDaemonSetNamespace:  common.KubeSystemNamespace,
+				RetinaDaemonSetName:       "retina-agent-win",
+				EbpfXdpDeamonSetNamespace: "install-ebpf-xdp",
+				EbpfXdpDeamonSetName:      "install-ebpf-xdp",
+				NonHpcAppNamespace:        "default",
```

Comment:
Suggest: can we please name this anything more descriptive than non-hpc? consider something like ebpf-test-pod or similar.